corehq/apps/data_interfaces/models.py

```diff
@@ -460,6 +460,16 @@ def to_json(self):
         data['id'] = self.id
         return data
 
+    def to_dict(self):
```

Comment:
Good point. I'm going to create docstrings for the two methods, but I'm mostly shying away from expected uses -- IMO methods should be self-contained so that it is up to the consumer how to use them.